client/components/select-dropdown/docs/example.jsx

```diff
@@ -43,10 +43,12 @@ var SelectDropdownDemo = React.createClass( {
 	},
 
 	render: function() {
-		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		const toggleButtonsText = this.state.compactButtons
+			? 'Normal Buttons'
+			: 'Compact Buttons';
 
 		return (
-			<div className="design-assets__group">
+			<div className="design-assets__group" style={ { height: 500 } }>
```

Comment:
Not a huge deal since this is an example, but could we try to avoid an inline style here?